### PR TITLE
Gv 3438 first guide page

### PR DESCRIPTION
**Pull request by Un-a**

### CODE REVIEW

File: packages/social-trader/src/components/guides/guides.helpers.ts [hunk 1]
```diff
@@ -1,34 +1,48 @@
...
+  return navGuides.reduce((acc: TGuide[], current) => {
     return [...acc, ...current.guides];
   }, []);
 };
 
-export const getCurrentGuide = (allGuides: Guide[], tab: string): Guide => {
+export const getCurrentGuide = (allGuides: TGuide[], tab: string): TGuide => {
   return safeGetElemFromArray(
     allGuides,
     guide => guide.canonicalName === tab.slice(1, tab.length)
```
Comment (pavelzubov) [inline packages/social-trader/src/components/guides/guides.helpers.ts:14]: image

File: packages/social-trader/src/pages/guides/guides.static-data.tsx [hunk 2]
```diff
@@ -0,0 +1,134 @@
+import ImageBaseElement from "components/avatar/image-base.element";
+import Deposit1 from "media/guides/guides-1/Deposit1.png";
+import Deposit2 from "media/guides/guides-1/Deposit2.png";
+import Deposit3 from "media/guides/guides-1/Deposit3.png";
+import Withdrawal1 from "media/guides/guides-1/Withdrawal1.png";
+import { WALLET_TOTAL_PAGE_ROUTE } from "pages/wallet/wallet.paths";
+import React from "react";
+
+export type TGuide = {
```
Comment (pavelzubov) [inline packages/social-trader/src/pages/guides/guides.static-data.tsx:9]: Use interface

File: packages/social-trader/src/pages/guides/guides.static-data.tsx [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+  canonicalName: string;
+  name: string;
+  content: JSX.Element;
+  linkInfo?: {
+    link: string;
+    label: string;
+  };
+};
+
+export type TNavGuide = {
```
Comment (pavelzubov) [inline packages/social-trader/src/pages/guides/guides.static-data.tsx:20]: Use interface

Comment (Un-a) [reply packages/social-trader/src/pages/guides/guides.static-data.tsx]: Done

File: packages/social-trader/src/pages/guides/guides.static-data.tsx (hunk 2 above)
Comment (Un-a) [reply packages/social-trader/src/pages/guides/guides.static-data.tsx]: Done

File: packages/social-trader/src/components/guides/guides.helpers.ts (hunk 1 above)
Comment (Un-a) [reply packages/social-trader/src/components/guides/guides.helpers.ts]: Done

**Pull request merged**